import-export-cli/cmd/login.go

```diff
@@ -119,8 +119,8 @@ func runLogin(store credentials.Store, environment, username, password string) e
 		return err
 	}
 
-	fmt.Println("Logged into", environment, "environment")
-	err = store.Set(environment, username, password, clientId, clientSecret)
+	fmt.Println("Logged into APIM in", environment, "environment")
```

Comment:
Should be corrected as fmt.Println("Logged into APIM in ", environment, "environment"), missing a space after "in"

Reply:
When `,` is used to concatenate strings, `Println` will add a space by default. https://play.golang.org/p/0lAnwRXpXrs

Reply:
no issues then. We can keep like this.